core/modules/utils/performance.js

```diff
@@ -68,14 +76,14 @@ Performance.prototype.log = function() {
 /*
 Wrap performance measurements around a subfunction
 */
-Performance.prototype.measure = function(name,fn) {
+Performance.prototype.measureFn = function(name, description,fn) {
```

Comment:
Accept and store description that will show in the Server timing

Reply:
We can't rename existing methods without considering the backwards compatibility impact. In this case, I don't think it is worth the disruption.